apidoc/Titanium/Filesystem/File.yml

```diff
@@ -176,19 +176,6 @@ methods:
         if this object doesn't identify a directory.
     returns: 
         type: Array<String>
-  - name: getParent
```

Comment:
I assume we actually need to deprecate this and not just remove it? Can we have a parent property that holds a Ti.Filesystem.File return value, but *also* have a getParent() method that returns the path of the parent as a String?

This is a breaking change, so we need to deprecate for one major version first... I assume there are apps out there relying on this getter existing and returning a string.

Reply:
I only removed it because the getter is generated automatically by using `- (NSString *)parent`. But I assume we could match Android's behavior here whereby `getParent` returns a file and `parent` a string. @infosia how does Windows behave here? Although I don't like that the getter of a property returns something different (oddly, I'd rather match the most-useful behavior). Maybe deprecate the difference on Android?